tests/unit/utils/test_unit.py

```diff
@@ -160,21 +173,32 @@ def test_base_verifier_run_action_on_unit(mock_run_action_on_units, model):
     mock_run_action_on_units.assert_called_with([unit], "test", True, None)
 
 
-@pytest.mark.parametrize("charm_url, exp_name", [
-    ("cs:focal/nova-compute-141", "nova-compute"),
-    ("cs:hacluster-74", "hacluster"),
-])
+@pytest.mark.parametrize(
+    "charm_url, exp_name",
+    [
+        ("cs:focal/nova-compute-141", "nova-compute"),
+        ("cs:hacluster-74", "hacluster"),
+    ],
+)
 def test_parse_charm_name(charm_url, exp_name):
     """Test function for parsing charm name from charm-ulr."""
     assert parse_charm_name(charm_url) == exp_name
 
 
-@pytest.mark.parametrize("charm_name, units", [
-    ("ceph-osd", [("ceph-osd", "ceph-osd/0")]),
-    ("ceph-osd", [("ceph-osd", "ceph-osd/0"), ("ceph-osd", "ceph-osd/1")]),
-    ("ceph-osd", [("ceph-osd", "ceph-osd-cluster-1/0"),
-                  ("ceph-osd", "ceph-osd-cluster-2/0")]),
-])
+@pytest.mark.parametrize(
+    "charm_name, units",
+    [
+        ("ceph-osd", [("ceph-osd", "ceph-osd/0")]),
+        ("ceph-osd", [("ceph-osd", "ceph-osd/0"), ("ceph-osd", "ceph-osd/1")]),
+        (
+            "ceph-osd",
+            [
+                ("ceph-osd", "ceph-osd-cluster-1/0"),
+                ("ceph-osd", "ceph-osd-cluster-2/0"),
+            ],
+        ),
+    ],
+)
```

Comment:
Another subjective point. I think that most of the `@pytest.mark.parametrize` decorators were more readable before `black`